src/test/java/Homework17.java

```diff
@@ -61,3 +61,32 @@ public static void addSongToPlayList() throws InterruptedException {
 
 
 
+import org.openqa.selenium.interactions.Actions;
+import org.testng.annotations.Test;
+
+public class Homework17 extends BaseTest {
+
+    @Test
+    public void addSongToPlaylist() throws InterruptedException {
+        navigateToPage();
+        login("demo@class.com", "te$t$tudent");
+        Thread.sleep(2000);
+        grabASong();
+
+    }
+
+    public void grabASong(){
+        WebElement song = driver.findElement(By.xpath("//article[@data-test='song-card']"));
+        WebElement playlist = driver.findElement(By.xpath("//section[@id='playlists']//li[5]"));
+
+        Actions acts = new Actions(driver);
+        acts.clickAndHold(song)
+                .release(playlist)
+                .build()
+                .perform();
+
+        playlist.click();
+//        acts.dragAndDrop(song, playlist).build().perform();
+
+    }
+}
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/210980130-17379fd9-5313-4279-916a-62797950e499.png)

![image](https://user-images.githubusercontent.com/104761482/210980373-23e9ddef-23e7-4f83-99c1-6625e5b3f49a.png)